### PR TITLE
Pinned notes

### PR DESCRIPTION
**Pull request by abhijeetekad**
Added pinned note functionality. 
Added note background colour.

### CODE REVIEW

**Comment by SanjayJatti** [comment]
Given functionality works fine. Just need to improve  CSS part.

File: src/Pages/MainPage/MainPage.jsx [hunk 1]
```diff
@@ -3,15 +3,21 @@ import "./MainPage.css";
 import { Navbar } from "../../Components/Navbar/Navbar.jsx";
 import { Footer } from "../../Components/Footer/Footer.jsx";
 import { Sidebar } from "../../Components/Sidebar/Sidebar.jsx";
+import { useSidebar } from "../../Context/SidebarContext";
 // import { NewNote } from "../../Components/NewNote/NewNote.jsx";
```
Comment (Manpreet-hub) [inline src/Pages/MainPage/MainPage.jsx]: remove commented code if not required.

File: src/Context/NoteContext.jsx [hunk 2]
```diff
@@ -48,9 +70,10 @@ const NoteProvider = ({ children }) => {
     note: [],
     recycle: [],
     archive: [],
+    pinNote: [],
   };
   const [stateNoteData, dispatchNoteData] = useReducer(noteFun, inititalValue);
-
+  console.log(stateNoteData);
```
Comment (Manpreet-hub) [inline src/Context/NoteContext.jsx]: remove console

**Review by Manpreet-hub — APPROVED** [review]
code looks clean, well structured and given functionality are also working fine.

File: src/Pages/MainPage/MainPage.jsx (hunk 1 above)
Comment (abhijeetekad) [reply src/Pages/MainPage/MainPage.jsx]: yes

File: src/Context/NoteContext.jsx (hunk 2 above)
Comment (abhijeetekad) [reply src/Context/NoteContext.jsx]: done

**Pull request merged**